localtileserver/custom/widgets.py

```diff
@@ -0,0 +1,76 @@
+import concurrent.futures
+import os
+import pathlib
+import uuid
+
+import anywidget
+import traitlets
+
+from localtileserver.client import TileClient
+
+THIS_DIR = pathlib.Path(__file__).parent.absolute()
+THREAD_EXECUTOR = concurrent.futures.ThreadPoolExecutor(max_workers=os.cpu_count())
+
+
+def handle_anywidget_command(widget, msg: str | list | dict, buffers: list[bytes]) -> None:
+    if not isinstance(msg, dict) or msg.get("kind") != "anywidget-command":
+        return
+
+    def target():
+        response, buffers = widget.get_tile(msg["msg"])
+        widget.send(
+            {
+                "id": msg["id"],
+                "kind": "anywidget-command-response",
+                "response": response,
+            },
+            buffers,
+        )
+
+    # t = threading.Thread(target=target)
+    # t.start()
+    THREAD_EXECUTOR.submit(target)
```

Comment:
I'm curious if you get much of a speedup with a threadpool executor. I would've thought the GIL would be held most of the time.

Reply:
Yes, I think I did notice a significant speed up  using multiple threads -- I'll have to make a comparison exmaple